src/bsk201903/connectMultipleClusters.md

```diff
@@ -14,14 +14,41 @@ _Should you create a single large cluster that spans multiple regions over a uni
 
 _Or should you have a lot of smaller clusters and find a way to manage and synchronise them?_
 
-Creating a single cluster that uses a unified network isn't recommended due to challenges such as:
+## A cluster to rule them all
 
-- **Distributing traffic using Kube-proxy.** Your requests could land in one region and being redirected to another
-- **Etcd being very picky when it comes to network latency.** etcd is so sensitive to network latency that even not running SSD disk could cause delays in reconciliations and slowdowns in your cluster.
+_Creating a single cluster that uses a unified network is a challenge._
```

Comment:
Storytime.
I thought _which_ was a more complicated version of _that_ because it required commas.
It turns out that using _that_ everywhere (like I do), isn't necessarily a smart move.
More on it: https://www.grammarly.com/blog/which-vs-that/
Given that my assumption was wrong, I'll review the article according to the link above and change accordingly.